index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang ="en">
+    <head>
+        <title>Daniel Hopton</title>
+        <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="https://cdn.rawgit.com/konpa/devicon/df6431e323547add1b4cf45992913f15286456d3/devicon.min.css">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <nav>
+            <div class="navwrapper">
+                <div class="navitem">
+                    <h4>Section Header</h4>
+                    <p>Section Description</p>
+                </div>
+                <div class="navitem">
+                    <h4>Section Header</h4>
+                    <p>Section Description</p>
+                </div>
+                <div class="navitem">
+                    <h4>Section Header</h4>
+                    <p>Section Description</p>
+                </div>
+            </div>
+        </nav>
+        <header>
+            <div class="headerwrapper">
+                <div class="headertext">
+                    <h1>FIRSTNAME SURNAME</h1>
+                    <h2>Job Title</h2>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+                </div>
+                <div class="badges">
+                    <i class="devicon-csharp-plain"></i>
+                    <i class="devicon-html5-plain"></i>
+                    <i class="devicon-css3-plain"></i>
+                </div>
+            </div>
+        </header>
+        <main class="showcase">
+            <section class="showcasewrapper">
```

Comment:
The idea of this section is that, in the grim darkness of the near future when this portfolio will have javascript/PHP on it you will be able to use the showcaseprev and showcasenext buttons to cycle through each project I've made, and the showcasetext and showcaseviewer's image and anchor reference will update accordingly. I believe this is how the main/section tags should be laid out?